src/metaswitch/clearwater/config_manager/move_config.py

```diff
@@ -506,9 +647,16 @@ def print_diff_and_syslog(config_1, config_2):
     additions = [line[2:] for line in difflines
                  if line.startswith("+ ") and len(line) > 2]
```

Comment:
Should we be trimming? I can't see any reason that we wouldn't want to ignore whitespace (but maybe we do and it's safer to avoid it?). I'm concerned about the case where we show a user a diff and they say "but I haven't modified line X!".

Reply:
When we print the lines out, we surround them in quotes - this will allow the user to see whitespace differences. As such, I don't think we need to do any trimming here.